gdal/ogr/ogrsf_frmts/shape/sbnsearch.c

```diff
@@ -1,12 +1,13 @@
+#define sanity_checks 1
```

Comment:
should be reverted

Reply:
Thanks for the catch.

Reply:
https://github.com/OSGeo/shapelib/commit/d9aed4b3254051cea1fe23a28ef5b873e0d926bb